bentoml/utils/validator/__init__.py

```diff
@@ -51,6 +51,10 @@
                     'region': {'type': 'string'},
```

Comment:
I saw `api_name` have `required: True`, do we also need that for instance_type and instance_count?

Reply:
both `instance type` and `instance count` has default value in the bentoml.cfg, that's probably why we dont have them as required.